unit_test/CMakeLists.txt

```diff
@@ -46,4 +50,7 @@ macro(CabanaMD_add_tests)
   endforeach()
 endmacro()
 
-CabanaMD_add_tests(MPI NAMES Integrator)
+CabanaMD_add_tests(NAMES Integrator Neighbor)
```

Comment:
Will you add the `MPI` case back here for tests that require multiple ranks?

Reply:
Yes, that should only be the Comm class